src/sections/target/ChemicalProbes/index.js

```diff
@@ -2,8 +2,7 @@ export const definition = {
   id: 'chemicalProbes',
   name: 'Chemical Probes',
   shortName: 'CP',
-  hasData: data =>
-    data.chemicalProbes?.rows.length || data.chemicalProbes?.probeminer,
+  hasData: data => data.chemicalProbes?.length > 0,
```

Comment:
From looking at the schema:
```
type Target {
  id: String!
  chemicalProbes: [ChemicalProbe!]!
  ...
}
```

we can omit the check for null of `chemicalProbes` since it's a required field in the GraphQL schema `data.chemicalProbes.length > 0`

Reply:
Thanks @mirandaio - good point. I've pushed the change.